client/src/pages/Home.js

```diff
@@ -1,11 +1,26 @@
 import { useEffect, useState } from "react";
 import { Link } from "react-router-dom";
 import MainTable from "../Component/MainTable";
+import SMSModal from "../Component/SMSModal";
 
 import "./Home.css";
 
+
+
 export function Home() {
 	const [message, setMessage] = useState("Loading...");
+	const [checkedCheckboxes, setChecked] = useState([]);
```

Comment:
- The naming convention for the setter is to take the name of the state and add `set` to the beginning, so here the setter should be `setCheckedCheckboxes`
- Optional: I was thinking about the naming of `checkedCheckboxes` and think a better one might be `selectedClientIds` since that's exactly what it holds. Up to you whether you can be bothered to update all the variables!